code/sql/schema.sql

```diff
@@ -1,96 +1,121 @@
 CREATE SCHEMA waze;
 
+CREATE TABLE waze.data_files
+(
+"id"                                SERIAL PRIMARY KEY NOT NULL,
+"start_time_millis"                 BIGINT NOT NULL,
+"end_time_millis"                   BIGINT NOT NULL,
+"start_time"                        TIMESTAMP,
+"end_time"                          TIMESTAMP,
+"date_created"                      TIMESTAMP,
+"date_updated"                      TIMESTAMP,
+"file_name"                         TEXT NOT NULL,
+"json_hash"                         uuid NOT NULL,
+UNIQUE ("start_time_millis", "json_hash")
+);
+
 CREATE TABLE waze.jams 
 (
-  "id"                  BIGINT PRIMARY KEY NOT NULL,
-  "uuid"                VARCHAR[500] NOT NULL,
-  "pub_millis"          BIGINT NOT NULL,
-  "start_node"          VARCHAR[500],
-  "end_node"            VARCHAR[500],
-  "road_type"           INTEGER,
-  "street"              VARCHAR[500],
-  "city"                VARCHAR[500],
-  "country"             VARCHAR[500],
-  "delay"               INTEGER,
-  "speed"               float4,
-  "length"              INTEGER,
-  "turn_type"           VARCHAR[500],
-  "level"               INTEGER,
-  "blocking_alert_id"   VARCHAR[500]
+  "id"                              SERIAL PRIMARY KEY NOT NULL,
```

Comment:
Part of the hashing proposal was to have the primary key on each of alerts/jams/irregularities be the actual hash.  So rather than having a serial id primary key we would have something like `record_hash varchar(40)`